CONTRIBUTING.md

```diff
@@ -33,6 +33,9 @@ You can run PHP tests with `vendor/bin/phpunit` and JavaScript tests with `npm t
 
 If you need any help with this please don't hesitate to ask.
 
+> :warning: If your filesystem uses CRLF instead of LF you may run into some issues when running the tests
+> You can run `find . -type f -not -path "./vendor" -not -path "./node_modules" -exec grep -qIP '\r$' {} ';' -exec perl -pi -e 's/\r\n/\n/g' {} '+'` to restore files back to LF
+
```

Comment:
Hmm, I actually tried that when I had this issue but it didn't solve it, because PHP_EOL is `\n` in PHP WSL but files are still CRLF, it's weird but yeah

Also files are all CRLF when cloning but when editing they get changed to LF by .editorconfig, so you get fixtures with CRLF and core files with LF which also gives you this problem

Lines endings issues are always a pain so it can't hurt to have it ;)

Reply:
If you run `git config --global core.autocrlf=input` that should resolve the checkout issue right? Everything will be cloned as `LF` and tests will pass?

Reply:
Yes indeed (without the `=`) 🎊 I already tried `git config --global core.autocrlf no` but it didn't seem to do anything, so don't know what's up with that